drivers/mock/driver.go

```diff
@@ -455,3 +455,21 @@ func (d *Driver) ExecTask(taskID string, cmd []string, timeout time.Duration) (*
 	}
 	return &res, nil
 }
+
+func parseDuration(s string) time.Duration {
+	if s == "" {
+		return time.Duration(0)
+	}
+
+	// check if it's an int64
+	if v, err := strconv.ParseInt(s, 10, 64); err == nil {
+		return time.Duration(v)
+	}
+
+	// try to parse it as duration
+	if v, err := time.ParseDuration(s); err == nil {
+		return v
+	}
+
+	panic(fmt.Errorf("value is not a duration: %v", s))
```

Comment:
Let's return an error even if this is a mock driver. It might be useful to test error handling and it would be a shame to crash people's agent if they're testing it with a mock_driver task and make a typo.

(If you haven't used mock_driver in a task when testing agents manually: it's super handy when you're testing scheduling or parts of the client and don't care about the actual driver mechanics.)

Reply:
done